### ISSUE TITLE
Add more information about the project

### ISSUE THREAD

**Issue by rstm-sf** [SuperJMN-Zafiro/Zafiro#3]
Hello!

Please, add more information about the project. Because I do not know what it is

**Comment by SuperJMN** [comment]
You're absolutely right. I hope I have some time to add a good description of what this does. In general terms it's a set of libraries that are add very commonly used functions, including string operations, reflection, patterns, controls for Avalonia and Uno Platform...

Thanks for your interest!

**Comment by SuperJMN** [comment]
I've added a bit more information about the project.
